### PR TITLE
[ADD] surveys: created new 'dashboard' doc

### PR DESCRIPTION
**Pull request by ksc-odoo**
PROJECT TASK: https://www.odoo.com/web#id=3811228&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

- created a new doc to focus SOLELY on the first thing users see when they open the *Surveys* application: the dashboard
- adjusted the main Surveys toctree to add 'dashboard' on the menu

This is part of a continued effort to reorganize/compartmentalize the Surveys doc, and not only have them freshly rewritten and updated for 17 to coincide with the release of the new tutorials, but they will also be broken down in a similar fashion to the tutorials.

Following this, I will write new doc for how to create a Survey -- which will takeover the current content on create.rst (so as not to cause any conflicts with any other doc it may be linked with/to)

Then, I'll write a doc to cover everything about 'Questions' (including Question Types, Question Options, and Conditional Displays)

Followed by other docs for 'Live Sessions' and 'Survey Analysis'

### CODE REVIEW

**Comment by ksc-odoo** [comment]
This PR is ready for a Peer Review from you -- whenever you get a chance, @Felicious 🙏  Thanks!

File: content/applications/marketing/surveys/dashboard.rst [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+- Number of :guilabel:`Questions` in that particular survey
+- :guilabel:`Average Duration` of the survey (how long it typically takes a participant to complete)
+- Number of :guilabel:`Registered` participants for the survey
+- Number of times that particular survey has been :guilabel:`Completed`
+- Percentage and data bar showcasing how many people have :guilabel:`Passed` or become
+  :guilabel:`Certified`
+
+.. note::
+   If there is a *half-trophy* background image behind the survey name, that means that particular
+   survey is a *Certification*.
```
Comment (Felicious) [inline content/applications/marketing/surveys/dashboard.rst:45]:
```suggestion
   A half-trophy background image behind the survey name indicates that the survey is a *Certification*.
```

File: content/applications/marketing/surveys/dashboard.rst [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+- :guilabel:`Share`: click to reveal a :guilabel:`Share a Survey` pop-up form that can be used to
+  invite potential participants to the survey - complete with a :guilabel:`Survey Link` that can be
+  copied and sent to potential participants, and a :guilabel:`Send by Email` toggle switch.
+
+   .. image:: dashboard/share-survey-popup.png
+      :align: center
+      :alt: The 'Share a Survey' pop-up window that appears in the Odoo Surveys application.
+
+  When the :guilabel:`Send by Email` toggle is in the 'on' position, indicated by a green switch,
```
Comment (Felicious) [inline content/applications/marketing/surveys/dashboard.rst:73]:
```suggestion
  When the :guilabel:`Send by Email` toggle is 'on' (green switch),
```

File: content/applications/marketing/surveys/dashboard.rst [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+  additional fields appear, in which :guilabel:`Recipients` and a :guilabel:`Subject` can be added
+  to the email. Below that, a dynamic email template, complete with a :guilabel:`Start
+  Certification` button appears, which can also be modified.
```
Comment (Felicious) [inline content/applications/marketing/surveys/dashboard.rst:76]:
```suggestion
  Certification` button is displayed, which can be modified as needed.
```

File: content/applications/marketing/surveys/dashboard.rst [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+
+  Once modifications are complete, click :guilabel:`Send` to send that email invite to all the email
+  addresses/contacts listed in the :guilabel:`Recepients` field.
```
Comment (Felicious) [inline content/applications/marketing/surveys/dashboard.rst:79]:
```suggestion
  addresses/contacts listed in the :guilabel:`Recipients` field.
```

File: content/applications/marketing/surveys/dashboard.rst [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+
+.. image:: dashboard/share-survey-popup-email-toggle.png
+   :align: center
+   :alt: The 'Share a Survey' pop-up window in Odoo Surveys with the Send by Email toggled on.
+
+.. important::
+   The :guilabel:`Share` button results in an error if the survey line that it is on has zero
+   questions.
```
Comment (Felicious) [inline content/applications/marketing/surveys/dashboard.rst:87]:
```suggestion
   Clicking the :guilabel:`Share` button results in an error if the survey line has zero
   questions.
```

**Review by Felicious — APPROVED** [review]
Amazing job with this doc, @ksc-odoo ! I just had some super minor wording suggestions, and after you take a look at those, feel free to send this along to the next stage of review!

**Comment by ksc-odoo** [comment]
Thanks for the feedback, @Felicious -- I made all your suggested edits. This one is now ready for a Tech Review from you, @samueljlieber -- whenever you get a chance. Thanks! 🙏

File: content/applications/marketing/surveys/dashboard.rst [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+- Number of :guilabel:`Questions` in that particular survey
+- :guilabel:`Average Duration` of the survey (how long it typically takes a participant to complete)
+- Number of :guilabel:`Registered` participants for the survey
+- Number of times that particular survey has been :guilabel:`Completed`
+- Percentage and data bar showcasing how many people have :guilabel:`Passed` or become
+  :guilabel:`Certified`
+
+.. note::
+   A half-trophy background image behind the survey name indicates that the survey is a
+   *Certification*.
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/dashboard.rst:45]: Suggesting to move this note below the bullet points since this is a minor visual cue that is added to Certification surveys, whereas the "Certified" percentage is a stronger visual cue that the record is a Certification survey
```suggestion
```

File: content/applications/marketing/surveys/dashboard.rst [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+
+.. note::
+   The :guilabel:`Passed` percentage and bar **only** appears when a *Required Score* has been
+   configured for that particular survey.
+
+   The :guilabel:`Certfied` percentage and bar **only** appears if that particular survey has the
+   *Is a Certification* option enabled on the survey form.
+
+   If neither :guilabel:`Passed` nor :guilabel:`Certified` appear on the line, that indicates the
+   survey is without a *Required Score* and was not enabled with the *Is a Certification* option.
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/dashboard.rst:55]: Indent to align with it's relevant bullet point and typo fix
```suggestion
  .. note::
     The :guilabel:`Passed` percentage and bar **only** appears when a *Required Score* has been
     configured for that particular survey.

     The :guilabel:`Certified` percentage and bar **only** appears if that particular survey has the
     *Is a Certification* option enabled on the survey form.

     If neither :guilabel:`Passed` nor :guilabel:`Certified` appear on the line, that indicates the
     survey is without a *Required Score* and was not enabled with the *Is a Certification* option.
```

File: content/applications/marketing/surveys/dashboard.rst [hunk 8]
```diff
@@ -0,0 +1,178 @@
...
+
+- Number of :guilabel:`Courses` related to that survey, which **only** appears if more than one
+  course has been created and attached to a single survey
+
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/dashboard.rst:59]: Move admonition
```suggestion

.. note::
   A half-trophy background image behind the survey name indicates that the survey is a
   *Certification*.

```

File: content/applications/marketing/surveys/dashboard.rst [hunk 9]
```diff
@@ -0,0 +1,178 @@
...
+Those buttons are as follows:
+
+- :guilabel:`Share`: click to reveal a :guilabel:`Share a Survey` pop-up form that can be used to
+  invite potential participants to the survey - complete with a :guilabel:`Survey Link` that can be
+  copied and sent to potential participants, and a :guilabel:`Send by Email` toggle switch.
+
+   .. image:: dashboard/share-survey-popup.png
+      :align: center
+      :alt: The 'Share a Survey' pop-up window that appears in the Odoo Surveys application.
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/dashboard.rst:71]: This image is not needed since the following displays what this image shows, as well as the advanced configuration.

Also update terminology, this "pop-up" is called a "modal" :)
```suggestion
- :guilabel:`Share`: click to reveal a :guilabel:`Share a Survey` modal that can be used to invite
  potential participants to the survey - complete with a :guilabel:`Survey Link` that can be copied
  and sent to potential participants, and a :guilabel:`Send by Email` toggle switch.
```

File: content/applications/marketing/surveys/dashboard.rst [hunk 10]
```diff
@@ -0,0 +1,178 @@
...
+
+  When the :guilabel:`Send by Email` toggle is 'on' (green switch), additional fields appear, in
+  which :guilabel:`Recipients` and a :guilabel:`Subject` can be added to the email. Below that, a
+  dynamic email template, complete with a :guilabel:`Start Certification` button appears, which can
+  be modified, as well, if needed.
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/dashboard.rst:76]: There are a few more fields that appear depending on the configuration of the email
```suggestion
  When the :guilabel:`Send by Email` toggle is active (green switch), additional fields appear, in
  which :guilabel:`Recipients`, :guilabel:`Additional Emails`, and a :guilabel:`Subject` can be
  added to the email.

  Below that, a dynamic email template, complete with a :guilabel:`Start Certification` button
  appears, which can be modified, as well, if needed.

  :guilabel:`Attachments` can be added to the email, as well as an :guilabel:`Answer deadline` can
  be set, if needed.
```

File: content/applications/marketing/surveys/dashboard.rst [hunk 11]
```diff
@@ -0,0 +1,178 @@
...
+
+  Once modifications are complete, click :guilabel:`Send` to send that email invite to all the email
+  addresses/contacts listed in the :guilabel:`Recipients` field.
+
+.. image:: dashboard/share-survey-popup-email-toggle.png
+   :align: center
+   :alt: The 'Share a Survey' pop-up window in Odoo Surveys with the Send by Email toggled on.
+
+.. important::
+   Clicking the :guilabel:`Share` button results in an error if the survey line has zero questions.
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/dashboard.rst:86]: It may be helpful to tip where users can edit the default **Mail Template**, as well as note the conditional display of fields depending on config :)
```suggestion
  .. tip::
     The default :guilabel:`Mail Template` for survey invites can be edited by navigating to
     :menuselection:`Settings --> Technical --> Email Templates` and searching for `Survey: Invite`.

  .. note::
     The :guilabel:`Send by Email` toggle switch is **not** present when the survey line has zero
     questions.

     The :guilabel:`Survey Link` only appears when the survey's *Access Mode* is set to *Anyone with
     the link*.

     The :guilabel:`Additional Emails` field only appears when the survey's *Require Login* field is
     **not** active.
```

File: content/applications/marketing/surveys/dashboard.rst [hunk 12]
```diff
@@ -0,0 +1,178 @@
...
+
+- :guilabel:`Test`: click to take a test version of the survey in a new tab, from the point-of-view
+  of a survey participant, in order to check for any errors or inconsistences.
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/dashboard.rst:89]: Typo
```suggestion
  of a survey participant, in order to check for any errors or inconsistencies.
```

File: content/applications/marketing/surveys/dashboard.rst [hunk 13]
```diff
@@ -0,0 +1,178 @@
...
+
+- :guilabel:`See results`: click to reveal a new tab showcasing detailed metrics and graphical
+  representations of all survey participants, questions, and responses for deeper analysis.
+
+- :guilabel:`Start Live Session`: click to initiate a *Live Session* survey, and reveal the *Session
+  Manager* window in a new tab. This button is **not** present for surveys that have enabled the
+  *Is a Certification* option on the survey form.
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/dashboard.rst:96]: Remove unnecessary formatting and update to casing to "Live session" as it is on the Odoo UI on Survey backend
```suggestion
- :guilabel:`Start Live Session`: click to initiate a *Live session* survey, and reveal a session
  manager window in a new tab. This button is **not** present for surveys that have enabled the
```

File: content/applications/marketing/surveys/dashboard.rst [hunk 14]
```diff
@@ -0,0 +1,178 @@
...
+
+- :guilabel:`End Live Session`: click to end a *Live Session* survey that has been officially
+  started. This button option **only** appears on survey lines that have previously initiated a
+  *Live Session*.
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/dashboard.rst:100]:
```suggestion
- :guilabel:`End Live Session`: click to end a *Live session* survey that has been officially
  started. This button option **only** appears on survey lines that have previously initiated a live
  session.
```

File: content/applications/marketing/surveys/dashboard.rst [hunk 15]
```diff
@@ -0,0 +1,178 @@
...
+
+Above the buttons that are located to the far-right of the survey lines, there is a mini
+*Configuration* menu that **only** appears when the cursor hovers over that particular line. This
+menu is represented by a :guilabel:`⋮ (three dots)` icon.
+
+When the :guilabel:`⋮ (three dots)` icon is clicked, a drop-down menu with some
+configuration-related options appear.
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/dashboard.rst:107]: Suggesting to restructure this sentence
```suggestion
Above the buttons that are located to the far-right of the survey lines, a :guilabel:`⋮ (three
dots)` icon appears when the cursor hovers over that particular line. When the :guilabel:`⋮ (three
dots)` icon is clicked, a drop-down menu with some configuration-related options appear:
```

File: content/applications/marketing/surveys/dashboard.rst [hunk 16]
```diff
@@ -0,0 +1,178 @@
...
+
+.. image:: dashboard/three-dot-dropdown.png
+   :align: center
+   :alt: The three-dot drop-down menu of options that appears on the Odoo Surveys dashboard.
+
+The options are:
+
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/dashboard.rst:114]: Suggesting to move this image below the bullet points and just jump right into the list
```suggestion
```

File: content/applications/marketing/surveys/dashboard.rst [hunk 17]
```diff
@@ -0,0 +1,178 @@
...
+- :guilabel:`Edit Survey`: when clicked, Odoo reveals the survey form for that particular survey,
+  which can then be modified in a number of different ways.
+- :guilabel:`Share`: when clicked, Odoo reveals the :guilabel:`Share a Survey` pop-up form that can
+  be used to invite potential participants to the survey.
+- :guilabel:`Delete`: when clicked, Odoo presents a pop-up window, wherein the user **must** confirm
+  they want to delete the survey entirely, which they can do by clicking the :guilabel:`Delete`
+  button at the bottom of the pop-up window.
+- :guilabel:`Color`: users can opt to choose a color to add to the survey line on the dashboard for
+  added organizational purposes, if needed.
+
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/dashboard.rst:124]:
```suggestion

.. image:: dashboard/three-dot-dropdown.png
   :align: center
   :alt: The three-dot drop-down menu of options that appears on the Odoo Surveys dashboard.

```

File: content/applications/marketing/surveys/dashboard.rst [hunk 18]
```diff
@@ -0,0 +1,178 @@
...
+.. image:: dashboard/activities-view.png
+   :align: center
+   :alt: The activities view option located on the Odoo Surveys dashboard.
+
+Doing so reveals a table with rows and columns. The rows show the different surveys in the database,
+and the columns depict the various activity types.
+
+.. important::
+   The :guilabel:`New` button does **not** appear in this view.
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/dashboard.rst:174]: I don't feel that this is _important_ information, rather it is just a note. It may be helpful to note the use-case of the view rather than just what _cannot_ be done while on this view
```suggestion
.. note::
   A new survey cannot be created in this view, as it is solely for the purpose of creating and
   viewing scheduled activities.
```

File: content/applications/marketing/surveys/dashboard/activities-view.png
Comment (samueljlieber) [inline content/applications/marketing/surveys/dashboard/activities-view.png:1]: The text in this image is tough to read, and may be able to be retaken with more clarity and less width?

File: content/applications/marketing/surveys/dashboard/share-survey-popup.png
Comment (samueljlieber) [inline content/applications/marketing/surveys/dashboard/share-survey-popup.png:1]: Suggesting to remove this image

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @ksc-odoo! Awesome job on this brand new Surveys doc, it is coming along nicely! I have some suggestions and corrections for you, please see below. 

I'd also like to suggest using either this `dashboard.rst` content or `create.rst` content as the main `surveys.rst` content, unless you plan on writing overview content specifically for `surveys.rst`. This would help remove an "extra step" towards getting to content in the TOC, let me know your thoughts!

Once these are addressed, tag me again for another quick look. Looking forward to seeing more Surveys content! :) Thank you!

File: content/applications/marketing/surveys/dashboard.rst (hunk 9 above)
Comment (ksc-odoo) [reply content/applications/marketing/surveys/dashboard.rst:71]: will delete the image, but am disregarding the "modal" suggestion, per our recent decision re: "modal" vs. "pop-up..."

**Comment by ksc-odoo** [comment]
@samueljlieber Thanks for all the great feedback -- i really appreciate it 🙏 Just made all your suggested edits. Ready for another look, whenever you get a chance. Thanks!

File: content/applications/marketing/surveys/dashboard.rst [hunk 19]
```diff
@@ -0,0 +1,188 @@
...
+
+     The :guilabel:`Additional Emails` field only appears when the survey's *Require Login* field is
+     **not** active.
+
+- :guilabel:`Test`: click to take a test version of the survey in a new tab, from the point-of-view
+  of a survey participant, in order to check for any errors or inconsistencies.
+
+- :guilabel:`See results`: click to reveal a new tab showcasing detailed metrics and graphical
+  representations of all survey participants, questions, and responses for deeper analysis.
+
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/dashboard.rst:105]:
```suggestion
```

File: content/applications/marketing/surveys/dashboard.rst [hunk 20]
```diff
@@ -0,0 +1,188 @@
...
+- :guilabel:`Start Live Session`: click to initiate a *Live session* survey, and reveal a session
+  manager window in a new tab. This button is **not** present for surveys that have enabled the *Is
+  a Certification* option on the survey form.
+
```
Comment (samueljlieber) [inline content/applications/marketing/surveys/dashboard.rst:109]:
```suggestion
```

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @ksc-odoo 

At a quick glance I'm thinking this type of doc would serve better as body content for `surveys.rst` since we're covering UI and settings. Those directory parent files are more ideal for this, whereas the children articles underneath them serve better as specific flows— anything that produces a desired outcome.

Can you make the adjustment, please, and then I'll take another look at the content? Thanks!

**Comment by ksc-odoo** [comment]
Alrighty, @StraubCreative -- I moved the dashboard content to the body of the Surveys.rst file, per your request -- and made sure to move all the images, as well *and* deleted the original dashboard.rst file, so I believe this is ready for another look from you whenever you get a chance. Thanks! 👍
